tests/sqlparser_postgres.rs

```diff
@@ -5157,3 +5157,34 @@ fn parse_create_type_as_enum() {
         _ => unreachable!(),
     }
 }
+
+#[test]
```

Comment:
The tests we can move over to common as part of switching over from the dialect_of macro to a dialect method

Reply:
Have Done!